hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/scm/ResetDeletedBlockRetryCountSubcommand.java

```diff
@@ -41,25 +50,45 @@ public class ResetDeletedBlockRetryCountSubcommand extends ScmSubcommand {
 
   static class TransactionsOption {
     @CommandLine.Option(names = {"-a", "--all"},
-        description = "reset all expired deleted block transaction retry" +
+        description = "Reset all expired deleted block transaction retry" +
             " count from -1 to 0.")
     private boolean resetAll;
 
     @CommandLine.Option(names = {"-l", "--list"},
         split = ",",
-        description = "reset the only given deletedBlock transaction ID" +
+        paramLabel = "txId",
+        description = "Reset the only given deletedBlock transaction ID" +
             " list, e.g 100,101,102.(Separated by ',')")
     private List<Long> txList;
-  }
 
-  @CommandLine.ParentCommand
-  private ScmAdmin parent;
+    @CommandLine.Option(names = {"-i", "--in"},
+        description = "Use file as input, need to be JSON Array format and" +
+            " contains multi \"txID\" key.")
```

Comment:
Can you add an example of JSON file here, can be just in code comments.

Reply:
![image](https://user-images.githubusercontent.com/10106574/222421214-f773bd60-aa2b-4ebc-a5c2-c51a73459081.png)
Updated as above